src/cudamatrix/cu-kernels.h

```diff
@@ -186,6 +186,10 @@ inline void cuda_add_mat_mat_elements(dim3 Gr, dim3 Bl, float *data, const float
 inline void cuda_replace_value(int Gr, int Bl, float *v, int dim, float orig, float changed) {cudaF_replace_value(Gr, Bl, v, dim, orig, changed); }
 inline void cuda_div_rows_vec(dim3 Gr, dim3 Bl, float *mat, const float *vec_div, MatrixDim d) { cudaF_div_rows_vec(Gr,Bl,mat,vec_div,d); }
 inline void cuda_set_bias_params(int Gr, int Bl, float* v, const float* a, float param_1, float param_2, float param_3, int* flag, int dim) { cudaF_set_bias_params(Gr,Bl,v,a,param_1,param_2,param_3,flag,dim); }
+inline void cuda_copy_from_vec(int Gr, int Bl, double* v_out, const float* v_in, int dim) { cuda_copy_from_vec_df(Gr,Bl,v_out,v_in,dim); }
+inline void cuda_copy_from_vec(int Gr, int Bl, float* v_out, const double* v_in, int dim) { cuda_copy_from_vec_fd(Gr,Bl,v_out,v_in,dim); }
+inline void cuda_copy_from_vec(int Gr, int Bl, float* v_out, const float* v_in, int dim) { cuda_copy_from_vec_ff(Gr,Bl,v_out,v_in,dim); }
```

Comment:
We don't need _ff or _dd functions. Since we can easily use cudaMemcpy in CopyFromVec to copy the vector. We can just define them for _fd or _df cases!